src/Uno.UI.Runtime.Skia.Linux.FrameBuffer/FramebufferHost.cs

```diff
@@ -156,6 +157,10 @@ void CreateApp(ApplicationInitializationCallbackParams _)
 			Windows.UI.Core.CoreDispatcher.DispatchOverride = Dispatch;
 			Windows.UI.Core.CoreDispatcher.HasThreadAccessOverride = () => _isDispatcherThread;
 
+			// We do not have a display timer on this target, we can use
+			// a constant timer.
+			CompositionTarget.UseGenericTimer = true;
```

Comment:
Should we have an external `CompositionTragetTimer` trigger instead of having a mix of internal timer and external trigger?